src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -419,12 +440,38 @@ protected boolean shouldActuallySendEmail(NotificationType notificationType, Str
     } else if (notificationType == NotificationType.PLACEMENT_ROLLOUT_2024_CORRECTION) {
       actuallySendEmail = inWhitelist
           || messagingControllerService.isValidRecipient(personId, MessageType.EMAIL);
+
+    } else if (notificationType == PROGRAMME_POG_MONTH_12) {
+
+      LocalDate now = LocalDate.now();
+      actuallySendEmail = inWhitelist
+          || (messagingControllerService.isValidRecipient(personId, MessageType.EMAIL)
+          && now.isAfter(POG_EPOCH));
     }
-    log.debug("Actually send email [{}]: for person {} and entity {} and notification {}",
+
+    log.info("Actually send email [{}]: for person {} and entity {} and notification {}",
         actuallySendEmail, personId, tisReferenceId, notificationType);
     return actuallySendEmail;
   }
 
+  /**
+   * Determine if the POG email should be stored (scheduled), ignoring whether we are currently
+   * before or after the POG_EPOCH.
+   *
+   * @param notificationType The notification type. Non-POG notification types will return false.
+   * @param personId         The person Id.
+   * @return True if the POG email should be scheduled, false otherwise.
+   */
+  protected boolean shouldStorePogEmail(NotificationType notificationType, String personId) {
```

Comment:
The point of this is to allow us to store the scheduled POG notifications; whether they will then be sent is determined at the point they are triggered, viz. 
```
actuallySendEmail = inWhitelist
          || (messagingControllerService.isValidRecipient(personId, MessageType.EMAIL)
          && now.isAfter(POG_EPOCH));
```
 above